traced/ykrt/src/mt.rs

```diff
@@ -47,6 +51,7 @@ impl MTBuilder {
     pub fn new() -> Result<Self, Box<dyn Error>> {
         Ok(Self {
             hot_threshold: DEFAULT_HOT_THRESHOLD,
+            max_worker_threads: cmp::max(1, num_cpus::get() - 1),
```

Comment:
So if I see it right, `num_cpus::get()` return different values over time? For example if in Linux I do:

```
# echo 0 > /sys/devices/system/cpu/cpu6/online
```

Then I can disable a core.

So should we delay this query to the latest moment?

This is a pretty spacious bikeshed.

Reply:
> This is a pretty spacious bikeshed.

You said it ;)

Reply:
Shouldn't this use `cmp::min`? Otherwise there will be either 0 or 1 worker threads.

Reply:
Using `max` guarantees we get at least 1 worker thread; but if you have, say, 12 cores, you'll get 11 worker threads.

Reply:
Got it mixed up.